### PR TITLE
Added upgradekey to config

### PR DESCRIPTION
**Pull request by benfes**
This change adds the upgradekey setting to config.php

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -466,6 +466,7 @@
 
 $CFG->passwordpolicy = false;
 $CFG->tool_generator_users_password = '12345';
+$CFG->upgrade = 'upgrade';
```
Comment (scq) [inline config.php:469]: I wonder if we should just make it an empty string -- I tried that locally and it meant I could just hit the button without entering anything.

Comment (scq) [reply config.php:469]: Also, I think this has to be `$CFG->upgradekey` rather than `$CFG->upgrade`.

Comment (chris-snyder-totara) [reply config.php:469]: Correct on both counts.

Comment (benfes) [reply config.php]: Nice catch, thank you.

**Review by scq — APPROVED** [review]

**Pull request merged**